scripts/testSetup.ts

```diff
@@ -115,6 +115,7 @@ export const getCustomNetworks = async (
     partnerChainID: l1NetworkInfo.chainId,
     retryableLifetimeSeconds: 7 * 24 * 60 * 60,
     nitroGenesisBlock: 0,
+    firstNitroBlock: 0,
```

Comment:
Do we need to set this? If we leave it at 0 it will treat all the txs hitting this and calling message events as nitro

Reply:
Yeh, I think it's fine to test on nitro